cluster/gce/util.sh

```diff
@@ -1318,11 +1318,6 @@ function create-nodes() {
   #TODO: parallelize this loop to speed up the process
   for ((i=1; i<=${NUM_MIGS}; i++)); do
     local group_name="${NODE_INSTANCE_PREFIX}-group-$i"
-    if [[ $i == ${NUM_MIGS} ]]; then
-      # TODO: We don't add a suffix for the last group to keep backward compatibility when there's only one MIG.
-      # We should change it at some point, but note #18545 when changing this.
-      group_name="${NODE_INSTANCE_PREFIX}-group"
-    fi
```

Comment:
I'm not sure we can actually do it due to backward compatibility. I think we still have clusters with such group and this may break those clusters, I think.

@gmarek

Reply:
AFAICT this change only affects `create-nodes`, the rest of the codebase seems to discover the group names dynamically so should not be affected.

Reply:
I found at least this one:
https://github.com/kubernetes/kubernetes/blob/7c04a684ae5350dc72a879ac168fd7e3d082b4b1/hack/ginkgo-e2e.sh#L75

I'm not sure if that's the only thing that has to be fixed though - it would require a deeper code inspection to ensure this won't break other things too.

Reply:
Ouch.